docs/DeveloperGuide.md

```diff
@@ -657,9 +661,9 @@ Guarantees:
     - Students are added to the student list in the respective tutorial groups upon successful command.
 
 MSS:
-    1. User chooses a csv file to upload.
-    2. User adds the csv file in the same folder as the JAR file.
-    3. Serenity reads the csv file.
+    1. User chooses an Excel file to upload.
```

Comment:
You can say "User chooses an Excel file of .xlsx type to upload" to be more specific